[VnV Plan] Add System Tests for Quotes (#204)
arkinmodi/project-sayyara: docs/VnVPlan/VnVPlan.tex

	      Output: A message saying there are no past quotes

	      Test Case Derivation: There are no past quotes to display

	      How test will be performed: The tester will request for past quotes through the user interface

	\item \textbf{FRT-BE8-2}

	      Control: Manual

	      Initial State: Customer Account with completed quotes having been made by the account

	      Input: Request past quotes

	      Output: A table with all the quotes

	      Test Case Derivation: There are past quotes to display

	      How test will be performed: The tester will request for past quotes through the user interface

	\item \textbf{FRT-BE8-3}

	      Control: Manual

	      Initial State: Shop Owner/Employee Account with no quotes having been made against the shop

	      Input: Request past quotes with quote ID, customer phone number, and/or customer name

	      Output: A message saying there are no past quotes

	      Test Case Derivation: There are no past quotes to display matching the inputted criteria

	      How test will be performed: The tester will request for past quotes through the user interface

	\item \textbf{FRT-BE8-4}

	      Control: Manual

> 	      Initial State: Shop Owner/Employee Account with completed quotes having been made against the shop

Reviewer: maybe "existing quote request", "closed"/"open" quotes for terminology -- same for below? what is a completed quote?

	      Output: A table with all the quotes

	      Test Case Derivation: There are past quotes to display matching the inputted criteria

	      How test will be performed: The tester will request for past quotes through the user interface

	\item \textbf{FRT-BE9-1}

	      Control: Manual

	      Initial State: Customer Account with completed quotes having been made by the account

	      Input: Request quote details

	      Output: The quote details

	      Test Case Derivation: Display the quote details requested

	      How test will be performed: The tester will request for quote details through the user interface

	\item \textbf{FRT-BE9-2}

	      Control: Manual

	      Initial State: Shop Owner/Employee Account with completed quotes having been made against the shop

	      Input: Request quote details

	      Output: The quote details

	      Test Case Derivation: Display the quote details requested

	      How test will be performed: The tester will request for quote details through the user interface

	\item \textbf{FRT-BE10-1}

	      Control: Manual

	      Initial State: Customer Account

	      Input: A filled in quote request form

> 	      Output: A quote created confirmation message

Reviewer: ```suggestion
	      Output: A confirmation messaging indicating that the quote was requested
```

	      Test Case Derivation: The system shall verify that the quote was created successfully

	      How test will be performed: The tester will create a quote through the user interface

	\item \textbf{FRT-BE10-2}

	      Control: Manual

	      Initial State: Shop Owner/Employee Account with a registered shop

> 	      Input: A quote is created against the shop

Reviewer: opened against
```suggestion
	      Input: A quote is requested from the shop
```

> 	      Output: A notification for the created quote

Reviewer: ```suggestion
	      Output: A notification for the open quote request
```

\wss{Include a blurb here to explain why the subsections below
	cover the requirements.  References to the SRS would be good here.}

\subsubsection{Quotes}

This section will contains tests covering the ``Quotes'' requirements defined in the
\href{https://github.com/arkinmodi/project-sayyara/blob/main/docs/SRS/SRS.pdf}{SRS}.

\begin{enumerate}

	\item \textbf{FRT-BE8-1}

	      Control: Manual

	      Initial State: Customer Account with no quotes having been made by the account

	      Input: Request past quotes

	      Output: A message saying there are no past quotes

	      Test Case Derivation: There are no past quotes to display

	      How test will be performed: The tester will request for past quotes through the user interface

	\item \textbf{FRT-BE8-2}

	      Control: Manual

	      Initial State: Customer Account with completed quotes having been made by the account

	      Input: Request past quotes

	      Output: A table with all the quotes

	      Test Case Derivation: There are past quotes to display

	      How test will be performed: The tester will request for past quotes through the user interface

	\item \textbf{FRT-BE8-3}

	      Control: Manual

> 	      Initial State: Shop Owner/Employee Account with no quotes having been made against the shop

Reviewer: ```suggestion
	      Initial State: Shop Owner/Employee Account with no quotes requests assigned to the shop
```

\wss{Include a blurb here to explain why the subsections below
	cover the requirements.  References to the SRS would be good here.}

\subsubsection{Quotes}

This section will contains tests covering the ``Quotes'' requirements defined in the
\href{https://github.com/arkinmodi/project-sayyara/blob/main/docs/SRS/SRS.pdf}{SRS}.

\begin{enumerate}

	\item \textbf{FRT-BE8-1}

	      Control: Manual

> 	      Initial State: Customer Account with no quotes having been made by the account

Reviewer: ```suggestion
	      Initial State: Customer Account with no quote requests initiated from the account
```

	      Input: Request past quotes

	      Output: A message saying there are no past quotes

	      Test Case Derivation: There are no past quotes to display

	      How test will be performed: The tester will request for past quotes through the user interface

	\item \textbf{FRT-BE8-2}

	      Control: Manual

> 	      Initial State: Customer Account with completed quotes having been made by the account

Reviewer: ```suggestion
	      Initial State: Customer Account with quotes requests which have been closed and were initiated by the account
```

	      Input: Request past quotes

	      Output: A table with all the quotes

	      Test Case Derivation: There are past quotes to display

	      How test will be performed: The tester will request for past quotes through the user interface

	\item \textbf{FRT-BE8-3}

	      Control: Manual

	      Initial State: Shop Owner/Employee Account with no quotes requests assigned to the shop

	      Input: Request past quotes with quote ID, customer phone number, and/or customer name

	      Output: A message saying there are no past quotes

	      Test Case Derivation: There are no past quotes to display matching the inputted criteria

	      How test will be performed: The tester will request for past quotes through the user interface

	\item \textbf{FRT-BE8-4}

	      Control: Manual

> 	      Initial State: Shop Owner/Employee Account with closed quotes having been made against the shop

Reviewer: ```suggestion
	      Initial State: Shop Owner/Employee Account with closed quotes requests that were assigned to the shop
```

	      Input: Request past quotes with quote ID, customer phone number, and/or customer name

	      Output: A table with all the quotes

	      Test Case Derivation: There are past quotes to display matching the inputted criteria

	      How test will be performed: The tester will request for past quotes through the user interface

	\item \textbf{FRT-BE9-1}

	      Control: Manual

> 	      Initial State: Customer Account with completed quotes having been made by the account

Reviewer: ```suggestion
	      Initial State: Customer Account with closed quote requests that were initiated by the account
```

	      Input: Request quote details

	      Output: The quote details

	      Test Case Derivation: Display the quote details requested

	      How test will be performed: The tester will request for quote details through the user interface

	\item \textbf{FRT-BE9-2}

	      Control: Manual

> 	      Initial State: Shop Owner/Employee Account with closed quotes having been made against the shop

Reviewer: ```suggestion
	      Initial State: Shop Owner/Employee Account with closed quote requests that were assigned to the shop
```
